### PR TITLE
Remove multiply and divide methods accepting double from ComplexNumbe…

### PR DESCRIPTION
**Pull request by jagdish-15**
# pull request

## Description
This pull request removes the multiply and divide methods that accept a double parameter from the ComplexNumber class in both the reference solution and the main.java file. These methods were deemed unnecessary since the primary focus of the exercise is to operate on complex numbers, and the existing tests do not validate these double overloads.

## Changes Made:
- Removed multiply(double factor) and divide(double divisor) methods from the ComplexNumber class.
- Updated the reference solution accordingly to ensure consistency.

## Rationale
The double overloads for multiplication and division were not accompanied by corresponding test cases in the canonical data, which could lead to situations where these methods are not adequately tested. By removing these methods, we streamline the class and enhance clarity, ensuring that the exercise focuses on complex number operations as intended.

## Next Steps
Please review the changes and let me know if further modifications are needed.
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java [hunk 1]
```diff
@@ -44,14 +46,6 @@ ComplexNumber conjugate() {
...
-    private ComplexNumber divide(double factor) {
-        return new ComplexNumber(real / factor, imaginary / factor);
-    }
-
-    private ComplexNumber multiply(double factor) {
-        return new ComplexNumber(factor * real, factor * imaginary);
+        return new ComplexNumber(Math.exp(real) * Math.cos(imaginary), Math.exp(real) * Math.sin(imaginary));
     }
-}
+}
```
Comment (kahgoh) [inline exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java:51]: Could you make check that there is an empty line at the end? Without it, [Checkstyle fails](https://github.com/exercism/java/actions/runs/11643029850/job/32423316068?pr=2852#step:4:680).

**Review by kahgoh — CHANGES_REQUESTED** [review]
Thanks for fixing the compile error @jagdish-15! However, there is a Checkstyle error. 

When fixing the issues from this review, feel free to put them in a commit on top of your current one and push to Github when ready. This should save you from having to close and opening PRs.

File: exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java (hunk 1 above)
Comment (jagdish-15) [reply exercises/practice/complex-numbers/.meta/src/reference/java/ComplexNumber.java:51]: Sure, I'll do it right now!

**Comment by jagdish-15** [comment]
"This commit passes all tests successfully. I appreciate any suggestions for further improvement!

**Review by kahgoh — APPROVED** [review]
Thanks! It looks good to me!

**Comment by jagdish-15** [comment]
Hi Kahgoh,

I realized that I submitted my GitHub handle as @jagdish-15 instead of jagdish-15 on Exercism, not knowing that the @ symbol shouldn't be included as this was my first PR ever, not just for Exercism but in general. I changed it to the correct handle after the PR was merged, but I didn’t receive the reputation points for my recent contributions. Could you please assist me in resolving this? Thank you for your help!

Best Regards,
Jagdish

**Comment by kahgoh** [comment]
I thought Exercism would pick it up after fixing your GitHub handle. You can check if it has by seeing if the PR appears on your [contributions](https://exercism.org/journey/reputation). If it hasn't, try removing the GitHub handle from Exercism and then putting it back again. If that hasn't worked, you might need to open up another thread on the forums.

**Comment by jagdish-15** [comment]
I tried everything, but it still didn’t work. The contribution isn’t showing up in the contributions section. I even tried removing and re-adding my GitHub handle, and temporarily changed my Exercism handle to match GitHub’s, but no luck. I'll open another thread on forums then.

**Pull request merged**